neuromation/cli/job.py

```diff
@@ -279,10 +279,11 @@ def job() -> None:
 @click.argument("job")
 @click.argument("cmd", nargs=-1, type=click.UNPROCESSED, required=True)
 @click.option(
-    "-t",
-    "--tty",
+    "-T",
```

Comment:
Please use `"-t/-T", "--tty/--no-tty"` form.
I'd like to keep `--tty` flag at least for backward compatibility but make it default.

Reply:
ok sure